surveyor.py

```diff
@@ -347,7 +361,7 @@ def survey(ctx, product_str: str = 'cbr') -> None:
         if opt.query:
             # if a query is specified run it directly
             log_echo(f"Running Custom Query: {opt.query}", log)
-            product.process_search(Tag('query'), base_query, opt.query)
+            product.process_search(Tag('query'), base_query, opt.json, opt.query)
```

Comment:
`opt.json` should be added to the kwargs that get passed when the product is initialized

```suggestion
            product.process_search(Tag('query'), base_query, opt.query)
```